dspace-server-webapp/src/main/java/org/dspace/app/rest/security/ClaimedTaskRestPermissionEvaluatorPlugin.java

```diff
@@ -49,7 +49,7 @@
     public boolean hasDSpacePermission(Authentication authentication, Serializable targetId,
                                  String targetType, DSpaceRestPermission permission) {
 
-        if (Constants.getTypeID(targetType) != Constants.CLAIMEDTASK) {
+        if (!StringUtils.equals("CLAIMEDTASK", targetType)) {
```

Comment:
I think we should not be hardcoding the type here again, and instead use `IndexableClaimedTask.TYPE` or similar

Reply:
Made a change here, but used ClaimedTaskRest.NAME to align more with the behavior in other permission evaluators.